bedrock/base/templates/cron-health-check.html

```diff
@@ -104,6 +104,12 @@ <h1 id="other-data">Other Site Data</h1>
         <td><a href="{{ server_info.db_file_url }}">{{ server_info.db_file_name }}</a></td>
       </tr>
       {% endif %}
+      {% if contentful_info %}
+      <tr>
+        <td title="Timestamp of the most recently updated ContentfulEntry">Latest Contentful sync</td>
+        <td>{{ contentful_info.latest_sync.strftime('%Y-%m-%d %H:%M:%S')}}</td>
```

Comment:
Will this throw an error if `contentful_info.latest_sync` is `None`?

Reply:
Also, most of our date handling on this page is done via time-since. Would that be okay too here you think? We could add titles to the elements with the full date stamp as well if that'd help. Maybe the time-since thing is just a personal preference though.

Reply:
I like the title being full datetime idea.

Reply:
There's also a `datetime` jinja filter in bedrock, so you could do `{{ contentful_info.latest_sync|datetime("%Y-%m-%d %H:%M:%S") }}`. It might be nice to pull that format string into a constant somewhere if we do use it elsewhere also.

Reply:
Thanks for the suggestions @pmac @robhudson - all make sense, and switching to the |datetime filter means we can handle `contentful_info.latest_sync` being the default of None, yeah because the filter does 
`return smart_text(t.strftime(fmt)) if t else ""`